### ISSUE TITLE
[BottomNavigationView] Badge accessibility not translated and not changeable

### ISSUE THREAD

**Issue by iAmMONK** [material-components/material-components-android#1898]
**Description:** The BottomNavigationView Badges are available only in English and there is no possibility to change it.

**Expected behavior:** Accessibility string resources should be translated or give developers possibility to implement own accessibility

**Source code:** The code snippet which is causing this issue. Please consider attaching a minimal sample app that reproduces the issue.

**Android API version:** Android 10

**Material Library version:** 1.2.1

**Device:** Pixel 3

The same applies for more components and I am wondering why is Accessibility setting hidden so deep sometimes. And developers has no chance to change it.

**Comment by wcshi** [comment]
The badge content description has now been translated:
https://github.com/material-components/material-components-android/tree/master/lib/java/com/google/android/material/badge/res

You can customize using:
https://github.com/material-components/material-components-android/blob/da2d8264d1644e85aa29ef1c1a37f306bc67d3ac/lib/java/com/google/android/material/badge/BadgeDrawable.java#L757-L765
